terraform/services/api-rds/main.tf

```diff
@@ -281,7 +288,7 @@ resource "aws_db_instance" "api" {
       var.app == "dpc" && var.env == "sbx") ? "4HR Daily Weekly Monthly" : "Daily Weekly Monthly"
     },
     var.app == "dpc" ? local.dpc_specific_tags : {}
-  )
+  ) : {}
```

Comment:
Like elsewhere, the reference to `data.default_tags.data_tags` should be removed along with the definition for the data source itself. Dropping `Name` from the greenfield is a good idea 👍 . I'm not actually sure if it's useful in legacy, either, but... that can come another day 😉